test/unit/filters.test.js

```diff
@@ -161,7 +161,7 @@ test('filter on body', t => {
       method: 'POST',
       body: jsonBuffer({
         query: `{
-        repositoryOwner(login: "search: "{\"username\": {\"$regex\": \"sue\"}, \"email\": {\"$regex\": \"sue\"}}"") {
+        repositoryOwner(login: "search: "{"username": {"$regex": "sue"}, "email": {"$regex": "sue"}}"") {
```

Comment:
wondering why `"` were escaped before...

Reply:
Think you're right, important for the test. Have brought it back and imported `gql-tag` to satisfy the linting.